vimode/src/vi.c

```diff
@@ -306,6 +306,14 @@ gboolean vi_notify_sci(SCNotification *nt)
 		}
 	}
 
+	if (nt->nmhdr.code == SCN_MARGINCLICK) {
```

Comment:
`{` on separate line.

Reply:
I've just merged #1338 which functionality-wise covers most of this PR but doesn't contain the `SCN_MARGINCLICK` part. Would you create a separate pull request containing it?